qcodes/instrument_drivers/QuTech/M2j.py

```diff
@@ -0,0 +1,83 @@
+from qcodes.instrument.base import Instrument
+from qcodes.utils.validators import Numbers
+import numpy as np
+
+try:
+    from spirack import M2j_module
+except ImportError:
+    raise ImportError(('The M2j_module class could not be found. '
+                       'Try installing it using pip install spirack'))
+
+
+class M2j(Instrument):
+
+    def __init__(self, name, spi_rack, module, **kwargs):
+        """
+        Qcodes driver for the M2j RF amplifier SPI-rack module.
+
+        Args:
+            name (str): name of the instrument.
+
+            spi_rack (SPI_rack): instance of the SPI_rack class as defined in
+                the spirack package. This class manages communication with the
+                individual modules.
+
+            module (int): module number as set on the hardware.
+
+        The gain and RF_level can only be set to the device, but not read from the device.
+
+        Parameters:
+            gain: Amplifier gain in dB, range 32 to 55 dB
+            RF_level: Measured RF power after amplification (not calibrated)
+        """
+        super().__init__(name, **kwargs)
+
+        self.m2j = M2j_module.M2j_module(spi_rack, module)
+
+        self.add_parameter('gain',
+                           label='gain',
+                           set_cmd=self._set_gain,
+                           unit='dB',
+                           vals=Numbers())
+
+        self.add_parameter('RF_level',
```

Comment:
Which of the two names reflects the value returned by the driver: "RF_level" or "power"? I have a feeling that the answer is "RF_level". If so, why do you call the method `_meas_power` instead of `_meas_rf_level`?